src/main/scala/contracts/token_collection_issuer_proxy/v1.0.0/ergoscript/Token_Collection_Issuer.es

```diff
@@ -0,0 +1,111 @@
+{
+
+    // ===== Contract Description ===== //
+    // Name: Token Collection Issuer Contract
+    // Description: Issuer box contract for the token sale collection, following the EIP-34 collection standard.
+    // Version: 1.0.0
+    // Author: Luca D'Angelo
+
+    // ===== Token Collection Mint Tx ===== //
+    // Description: This transaction mints the collection tokens.
+    // Data Inputs: None
+    // Inputs: Token Collection Issuer Box
+    // Context Variables: None
+    // Outputs: Token Collection Issuance Box
+
+    // ===== Box Registers ===== //
+    // R4: Int                              => Collection Standard Version
+    // R5: Coll[Coll[Byte]]                 => Collection Information
+    // R6: Coll[(Coll[Byte], Coll[Byte])]   => Social Media Information
+    // R7: Long                             => Minting Expiry Timestamp
+    // R8: Coll[(Coll[Byte], Coll[Byte])]   => Additional Information
+
+    // ===== Compile Time Constants ===== //
+    // _RarityPoolsAmount: Long
+    // _IsWhitelist: Boolean
+    // _MinerFee: Long
+
+    // ===== Context Extension Variables ===== //
+    val TokenColectionIssuanceContractBytes: Coll[Byte] = getVar[Coll[Byte]](0).get
+    val CollectionTokenVerboseName: Coll[Byte] = getVar[Coll[Byte]](1).get
+    val CollectionTokenDescription: Coll[Byte] = getVar[Coll[Byte]](2).get
+    val CollectionTokenNumberOfDecimals: Coll[Byte] = getVar[Coll[Byte]](3).get // Should be 0, encoded according to EIP-4
+    val CollectionTokenAssetType: Coll[Byte] = Coll(3.toByte, 2.toByte) // The asset type category should be Utility-Token (or NFT Artwork?)
+    val CollectionTokenPictureSHA256Hash: Coll[Byte] = Coll[Byte]() // Should just be empty Coll[Byte](), if Utility-Token
+    val CollectionTokenPictureLink: Coll[Byte] = Coll[Byte]() // Should just be empty Coll[Byte](), if Utility-Token
```

Comment:
The tx operator decides the context variables, so they can insert whatever they want